.github/workflows/scripts/dependabot-pr.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash -ex
+# Copyright The OpenTelemetry Authors
```

Comment:
We have a line break here in contrib scripts. Should we do the same in core?

Reply:
Done